service/awsconfig/v5/resource/loadbalancer/resource.go

```diff
@@ -6,8 +6,6 @@ import (
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/micrologger"
 	"github.com/giantswarm/operatorkit/framework"
-
-	awsservice "github.com/giantswarm/aws-operator/service/aws"
```

Comment:
Removed the aws service because it isn't needed.

Reply:
Nice! We will probably able to remove it completely medium term.